x-pack/plugins/security_solution/public/management/cypress/cypress_endpoint_serverless.config.ts

```diff
@@ -0,0 +1,79 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { defineCypressConfig } from '@kbn/cypress-config';
+
+// eslint-disable-next-line import/no-default-export
+export default defineCypressConfig({
```

Comment:
These new config seem to be very similar and only really change a few parameters. Wondering if we should instead have a `*base*` config that is shared across of these deplouyment specific configs - I think that would make management of them a bit better.

Reply:
thanks, great idea! done: ad1c3fe55aa0a5ac51551b603d8212c8cf65bcc3